index.html

```diff
@@ -418,6 +419,163 @@ <h2>Transferable MediaStreamTrack</h2>
       <p>The underlying source is supposed to be kept alive between the transfer and transfer-receiving steps, or as long as the data holder is alive.
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
+    <h3>MediaStreamTrack frame stats</h3>
+    <div>
+      <p>
+        On camera and screenshare tracks, frame counters allow the application
+        to tell what the frame rate is, which may be lower than the target
+        {{MediaTrackSettings/frameRate}}. For example, if the track is sourced
+        from a camera then the production of frames could be slowed down if it's
+        dark or frames could be dropped if the system is CPU starved. This could
+        impact the total number of frames produced by the source and impact how
+        many frames are delivered, discarded or dropped for other reasons.
+      </p>
+    </div>
+    <div>
+      <pre class="idl"
+>partial interface MediaStreamTrack {
+  Promise&lt;MediaTrackFrameStats&gt; getFrameStats();
+};
+</pre>
+    </div>
+    <div>
+      <p>If a {{MediaStreamTrack}} is sourced from <code>getUserMedia()</code>
+      or <code>getDisplayMedia()</code>, the user agent is required to count
```

Comment:
Note: I was unable to get `{{MediaDevices/getDisplayMedia()}}` to link correctly (despite by best attempt at updating the terminology section) so I used `<code>getDisplayMedia()</code>` instead. Unless someone knows how to fix this, I suggest we either create a follow-up issue "Fix getDisplayMedia() reference" or that I rephrase this as "sourced from a camera or screenshare"